.gitignore

```diff
@@ -1,15 +1,9 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
 # dependencies
 /node_modules
-/.pnp
-.pnp.js
-
-# testing
-/coverage
+/package-lock.json
```

Comment:
I'm removing the lock file, as `react-twitch-components` is a library.
`package-lock.json` files are never included in npm packages, so you can't make sure that the dependencies you've locked on are the same your users will get. As a library it needs to make sure it works with the latest dependencies within the version range specified.

Reply:
Ah that makes sense!